### ISSUE TITLE
COLP Review

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-colp#188]
- [x] update UIDs in modification table
- [ ] Missing 1 record?
- [x] Misalignment between CD and `Borough`
Misalignment is valid.  Records where CD is NULL are valid because these records don't have a CD value in PLUTO
- [x] 3 records have a NULL `Category` value
Valid
- [x] 56 records have a NULL `EXPANDCAT` value
Valid
- [x] Modifications not applied
Valid - these uids no longer exist in COLP

**Comment by lseirup** [comment]
There are 483 records for which IPIS has a CD, but we blanked it out because MapPLUTO has a null value. 

The problem is with MapPLUTO. Version 21v1, which we used for the previous version of COLP, had only 24 null CD values. 21v2 and 21v3 have over 700 null CD values, mostly on the shoreline. Did we start using the clipped CD boundaries in 21v2 to backfill missing CD values?

For COLP, I don't want to wipe out valid CD values coming from IPIS. Can we keep the IPIS value for CD if PLUTO has a null value?
